e2e/test-util.ts

```diff
@@ -127,36 +127,31 @@ export async function createVFolderAndVerify(
   folderName: string,
   usageMode: 'general' | 'model' = 'general',
   type: 'user' | 'project' = 'user',
-  permission: 'rw' | 'ro' | 'wd' = 'rw',
+  permission: 'rw' | 'ro' = 'rw',
 ) {
-  const permissionMap = {
-    rw: 'Read & Write',
-    ro: 'Read only',
-    wd: 'Delete',
-  };
   await navigateTo(page, 'data');
 
-  await page.getByRole('button', { name: 'plus Add' }).click();
+  await page.getByRole('button', { name: 'Create Folder' }).nth(1).click();
   await page.getByRole('textbox', { name: 'Folder name' }).fill(folderName);
+
   // select parsed parameters in create modal form
-  await page.getByRole('radio', { name: usageMode }).click();
-  await page.getByRole('radio', { name: type }).click();
-  await page.getByRole('radio', { name: permissionMap[permission] }).click();
+  await page.getByTestId(`${usageMode}-usage-mode`).click();
+  await page.getByTestId(`${type}-type`).click();
+  await page.getByTestId(`${permission}-permission`).click();
 
   await page.getByRole('button', { name: 'Create', exact: true }).click();
   await page.reload();
-  const nameInput = page
-    .locator('#general-folder-storage vaadin-grid-cell-content')
-    .filter({ hasText: 'Name' })
-    .locator('vaadin-text-field')
-    .nth(1)
-    .locator('input');
-  await nameInput.click();
-  await nameInput.fill(folderName);
+  await page.getByTestId('vfolder-filter').locator('div').nth(2).click();
+  await page.getByRole('option', { name: 'Name' }).locator('div').click();
+  await page.locator('#rc_select_8').click();
+  await page.locator('#rc_select_8').fill(folderName);
```

Comment:
There is no difference in performance, but for the input locator, the text will be filled even if you just use fill without clicking.